src/index.js

```diff
@@ -94,6 +52,13 @@ class HCaptcha extends React.Component {
 
       const isApiReady = typeof this._hcaptcha !== 'undefined';
 
+      this.sentryHub = initSentry(this.props.sentry, scopeTag);
+
+      this.sentryHub?.addBreadcrumb({
```

Comment:
better to replace optional chaining here with good old if

Reply:
Not a fan of optional chaining or are you concerned that `this.sentryHub` is null?

Reply:
let's keep it readable and plan to handle `sentry: false` flag in loader by making sentry wrapper not return null